locale/en/knowledge/child-processes/how-to-spawn-a-child-process.md

```diff
@@ -11,29 +11,49 @@ layout: knowledge-post.hbs
 
 If you find yourself wishing you could have your Node.js process start another program for you, then look no further than the `child_process` module.
 
-The simplest way is the "fire, forget, and buffer" method using `child_process.exec`.  It runs your process, buffers its output (up to a default maximum of 200kb), and lets you access it from a callback when it is finished. Let us take a look at an example:
+The simplest way is the "fire, forget, and buffer" method using `child_process.exec`.  It runs your process, buffers its output (up to a default maximum of 200kb), and lets you access it from a callback when it is finished. 
 
-     var childProcess = require('child_process'),
-         ls;
+*Note that the examples you will see in this article are all Linux-based. On Windows, you need to switch these commands with their Windows alternatives.*
 
-     ls = childProcess.exec('ls -l', function (error, stdout, stderr) {
-       if (error) {
-         console.log(error.stack);
-         console.log('Error code: '+error.code);
-         console.log('Signal received: '+error.signal);
-       }
-       console.log('Child Process STDOUT: '+stdout);
-       console.log('Child Process STDERR: '+stderr);
-     });
+Let us take a look at an example:
 
-     ls.on('exit', function (code) {
-       console.log('Child process exited with exit code '+code);
-     });
+```js
+const { exec } = require('child_process');
 
-NODE PRO TIP: `error.stack` is a stack trace to the point that the [Error object](/what-is-the-error-object) was created.
+const ls = exec('ls -l', function (error, stdout, stderr) {
+  if (error) {
+    console.log(error.stack);
+    console.log('Error code: '+error.code);
+    console.log('Signal received: '+error.signal);
+  }
+  console.log('Child Process STDOUT: '+stdout);
+  console.log('Child Process STDERR: '+stderr);
+});
+
+ls.on('exit', function (code) {
+  console.log('Child process exited with exit code '+code);
+});
+```
+
+NODE PRO TIP: `error.stack` is a stack trace to the point that the [Error object](https://github.com/nodejs/nodejs.org/blob/master/locale/en/knowledge/errors/what-is-the-error-object.md) was created.
```

Comment:
Link should be `/en/knowledge/errors/what-is-the-error-object/`.

Reply:
https://nodejs.org/en/knowledge/errors/what-is-the-error-object/ works, please revert your last commit to `/en/knowledge/errors/what-is-the-error-object/`. 😉